hackman_payments/management/commands/import_payments.py

```diff
@@ -13,16 +12,9 @@ class Command(BaseCommand):
 
     def handle(self, *args, **kwargs):
         p = subprocess.run([
-            'git', '-C', 'dsl-accounts/', 'pull', '-f', 'origin', 'master'
+            'curl', 'curl https://dimsumlabs.github.io/dsl-accounts-pages/paym'
```

Comment:
We need to save this file to disk so the system can start up properly even without an internet connection.

Reply:
Good point, try this next patch then